masterlist.yaml

```
@@ -18868,6 +18868,30 @@ plugins:
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 14
 
+  - name: 'Ryn''s Farms.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/72305/'
+        name: 'Ryn''s Farms'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Sewers'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Skyrim Sewers.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Rift Watchtower NavFix'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Riften Watchtower Nav Fix.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Ryn''s Standing Stones](https://www.nexusmods.com/skyrimspecialedition/mods/64969/)'
```

You don't need to link to the patched mod in a patch message; presumably users already know where they got it from:

```suggestion
          - 'Ryn''s Standing Stones'
```